misc/container-metadata-file-validator/container-metadata-file-validator.go

```diff
@@ -0,0 +1,225 @@
+// Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
looks like this file is essentially the same as `misc/container-metadata-file-validator-windows/container-metadata-file-validator-windows.go` . is it possible to reuse?

Reply:
There is different validation for network on linux, which is more detailed. The 2 files approach is taken from the v3-task-metadata-validator. The files are built/managed differently. In addition, the task definitions themselves are a bit different (with windows including "windows" identifiers)